Framework/Algorithms/inc/MantidAlgorithms/CalculateCarpenterSampleCorrection.h

```diff
@@ -1,20 +1,13 @@
 #ifndef MANTID_ALGORITHM_MULTIPLE_SCATTERING_ABSORPTION_H_
 #define MANTID_ALGORITHM_MULTIPLE_SCATTERING_ABSORPTION_H_
 
-#include "MantidAPI/AlgorithmManager.h"
-#include "MantidAPI/AnalysisDataService.h"
 #include "MantidAPI/DataProcessorAlgorithm.h"
-#include "MantidAPI/WorkspaceGroup.h"
-#include "MantidAPI/WorkspaceUnitValidator.h"
 #include "MantidHistogramData/Points.h"
 #include <vector>
 
+using namespace Mantid::API;
```

Comment:
It's a small point but using declarations like this should never be in header files as they can spread around the quite easily and cause problems with ambiguity. Header files should always qualify names as you had before.